src/tasks/che.ts

```diff
@@ -554,4 +555,42 @@ export class CheTasks {
       }
     ]
   }
+
+  debugTask(flags: any): ReadonlyArray<Listr.ListrTask> {
+    return [
+      {
+        title: 'Find Che Server pod',
+        task: async (ctx: any, task: any) => {
+          const chePods = await this.kube.listNamespacedPod(flags.chenamespace, undefined, this.cheSelector)
+          if (chePods.items.length === 0) {
```

Comment:
Does it make sense to check if there are more than 1 item?
It's possible after rescaling when one pod is terminating and another is starting...
picking the first up could be not a straightforward thing to do.

Reply:
Yeap, but more than 1 server is possible case.